docs/source/sysadmin_guide/off_chain_settings.rst

```diff
@@ -0,0 +1,300 @@
+****************************************************
+Changing Off-chain Settings with Configuration Files
+****************************************************
+
+Each Sawtooth component, such as the validator or the REST API, can have an
+optional configuration file that controls the component's behavior. By default,
+Sawtooth does not install any configuration files.  However, Sawtooth provides
+example configuration files that can be customized for your system.
+
+This procedure explains how to use Sawtooth configuration files to change the
+following Sawtooth settings:
+
+* Host and port values (bind endpoints) for the validator, REST API, and
+  consensus proxy
```

Comment:
Consensus Proxy is an internal implementation detail of the validator and should not be mentioned at this level.

Reply:
@aludvik - Would the admin ever have to change the ``bind = [ .... consensus:` hostname and port?  (In a realistic scenario, that is.)

If yes, consensus (with or without the word "proxy") needs to be mentioned here.  If not, I'll delete this and say "don't change the consensus setting" in the `bind` step.

Reply:
(From aludvik via phone): Change to "consensus API".